backend/backend.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+
+apt-get update
+
+
+#need in swtpm 
+apt -y install autoconf-archive libcmocka0 libcmocka-dev procps iproute2 build-essential git pkg-config gcc libtool automake libssl-dev uthash-dev autoconf doxygen libjson-c-dev libini-config-dev libcurl4-openssl-dev libltdl-dev expect python3-yaml libglib2.0-dev dbus  libtasn1-6-dev libjson-glib-dev gawk socat libseccomp-dev uuid-dev udev libusb-1.0-0-dev libgnutls28-dev gnutls-bin
+
+
+#need in acs
+apt-get install -y libjson-c4  libjson-c-dev mariadb-server libmysqlclient-dev git make gcc wget 
+rm -rf /var/lib/apt/lists/*
+
+rm -rf /root/tpm2  /root/acs  /root/startup.sh /root/swtpm
+
+mkdir /root/tpm2 && \
+    cd /root/tpm2 && \
+    wget https://versaweb.dl.sourceforge.net/project/ibmtpm20tss/ibmtss1.6.0.tar.gz && \
```

Comment:
please use `wget -P /toot/tpm2 https://versaweb.dl.sourceforge.net/project/ibmtpm20tss/ibmtss1.6.0.tar.gz` to replace line 18 and 19